crates/wash-cli/src/down/mod.rs

```diff
@@ -109,14 +129,18 @@ pub async fn handle_down(cmd: DownCommand, output_kind: OutputKind) -> Result<Co
             return Ok(CommandOutput::new(out_text, out_json));
         } else {
             let wasmcloud_pid_file_path = install_dir.join(WASMCLOUD_PID_FILE);
-            tokio::fs::remove_file(&wasmcloud_pid_file_path)
+            // Failing to find the pid file is not an error that should prevent stopping other resources
+            if let Err(e) = tokio::fs::remove_file(&wasmcloud_pid_file_path)
                 .await
                 .with_context(|| {
                     format!(
                         "failed to find wasmcloud pid file [{}]",
                         wasmcloud_pid_file_path.display()
                     )
-                })?;
+                })
+            {
+                warn!("{}", e.to_string());
```

Comment:
```suggestion
                warn!("failed to remove wasmcloud PID file [{}]: {}", wasmcloud_pid_file_path.display(), e.to_string());
```

Reply:
The warn does include this information! It's added with the `with_context` call above. For example:
```
➜ RUST_LOG=warn cargo run -- down --purge all
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.28s
     Running `/Users/brooks/github.com/wasmcloud/wasmcloud/target/debug/wash down --purge all`
>>> ⡀⠀  Stopping wasmCloud ...                                                                                                   2024-06-07T13:56:40.989987Z  WARN wash_cli::down: failed to find wasmcloud pid file [/Users/brooks/.wash/downloads/wasmcloud.pid]

✅ wasmCloud hosts stopped successfully
❌ Could not stop wadm: No pidfile found at [/Users/brooks/.wash/downloads/wadm.pid]
✅ NATS Jetstream purged successfully
❌ NATS server did not stop successfully: No pidfile found for nats-server, assuming it's managed externally
🛁 wash down completed successfully
```

Reply:
Ahhh sorry that's right -- I was distracted by the existing message... it's a little wrong, it should say it failed to *delete* the file (there could be other reasons)